benefits/templates/admin/base_site.html

```diff
@@ -1,5 +1,5 @@
-{% extends "admin/base_site.html" %}
+{% extends "admin/base.html" %}
 
-{% block extrahead %}
-  {% include "admin/includes/favicon.html" %}
-{% endblock extrahead %}
```

Comment:
Originally had intended to delete this file entirely, but all of the Admin app won't get this new branding unless we override this here as well.

Reply:
I guess I don't fully understand the difference between admin/base.html and admin/base_site.html and why we need both?

But my main question is, if admin/base_site.html (this template) extends admin/base.html (our custom version of that template) -- then why do we need the same {% block branding %} here?

Reply:
I.e. maybe it is enough to just have this template `{% extends admin/base.html %}` and that's all?

Reply:
Deleting the `base_site.html` file entirely leads to this:
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/6fdeef9a-05f9-444e-b464-3e463e325bd1" />

Keeping `base_site.html` but taking out the `branding` override behaves as expected. The `branding` override inherits from our `base.html`.

Reply:
@thekaveman Fixed here: https://github.com/cal-itp/benefits/pull/2773/commits/1dca8396a9440c2dee46bb9e3bbf1ebdf739cdda